lib/Echidna/ABI.hs

```diff
@@ -62,21 +62,23 @@ encodeSig n ts = n <> "(" <> T.intercalate "," (map abiTypeSolidity ts) <> ")"
 genSize :: MonadGen m => m Int
 genSize = (8 *) <$> Gen.enum 1 32
 
-genWord256 :: MonadGen m => m Word256
-genWord256 = let w64 = Gen.word64 $ constant minBound maxBound in
-  liftM2 Word256 (liftM2 Word128 w64 w64) (liftM2 Word128 w64 w64)
-
 genAbiAddress :: MonadGen m => m AbiValue
 genAbiAddress = let w64 = Gen.word64 $ constant minBound maxBound in
   fmap AbiAddress . liftM2 Word160 Gen.enumBounded $ liftM2 Word128 w64 w64
 
+maxInteger :: Integer
+maxInteger = 2^(256 :: Integer)
+
+genUInt :: MonadGen m => m Integer
```

Comment:
I'm not sure these two are quite correct. Aren't 256 bit UInts in [0, 2^256 -1], and Ints in [-2^128, 2^128-1]?